src/stores/drawer.ts

```diff
@@ -1,13 +1,14 @@
 import { atom } from 'recoil';
+import ROUTING_PATHS from 'utils/constants/routingPaths';
 
 export const foodPartyCreateDrawerOpenState = atom<boolean>({
   key: 'foodPartyCreateDrawerOpen',
   default: false,
 });
 
-export const loginDrawerOpenState = atom<boolean>({
+export const loginDrawerState = atom<{ isOpen: boolean; urlAfterLogin: string }>({
```

Comment:
p5: RedirectUrlType으로 잘 정의해주셨는데 그걸 사용해보는 건 어떨까요?

```suggestion
import { RedirectUrlType } ...

export const loginDrawerState = atom<{ isOpen: boolean; } & RedirectUrlType>({
```

Reply:
대박 ! 감사합니다 👍 
(타입 정의 전에 작성했던 코드라 놓친 거 같네요)

Reply:
반영완료 [5397a16](https://github.com/prgrms-web-devcourse/Team-DarkNight-Kkini-FE/pull/184/commits/5397a1602d341303dec25c3d9846dc4dd45dc8b3)